README.md

```diff
@@ -55,7 +55,7 @@ Start an anger instance against a given target.
     * `connections`: The number of connections to maintain to server. Each is an individual nes client.
     * `senders`: The number of connections that should be senders.
     * `responses`: The number of overall expected responses to be recieved during this run.
-    * `auth`: An `Object` passed to the nes client for authentication. [reference.][nes-auth]
+    * `auth`: A `Function` passed to the nes client for authentication. This is passed the `client` and `index` of that client as params. Must return the auth object options. [reference.][nes-auth]
```

Comment:
👍  on supporting both objects and functions. The reason for having the index is mainly to pick next token from an array. I suppose I could to that by maintaining a counter external to anger though. If it's a `sender` I might want a specific token, that's why I think that would be useful.